packages/frontend/src/redux/slices/tokens/index.js

```diff
@@ -76,6 +76,24 @@ const fetchTokens = createAsyncThunk(
     }
 );
 
+const fetchToken = createAsyncThunk(
```

Comment:
I don't see any usages of this function

Reply:
This function will be used for fetching farm tokens from validator